doc/api/process.md

```diff
@@ -863,12 +863,14 @@ if (process.getuid) {
 }
 ```
 
-## process.hrtime()
+## process.hrtime([time])
```

Comment:
From other docs, it seems that brackets are used to indicate optional parameters, not their contents (c.f. https://nodejs.org/dist/latest-v6.x/docs/api/process.html#process_process_kill_pid_signal).  I call it out, though, for extra scrutiny because it might be confusing given the `[seconds, nanoseconds]` notation that follows shortly thereafter.